libs/shared/src/services/http-client/api.client.ts

```diff
@@ -14,47 +12,89 @@ export interface IPaginatedResponse<T = unknown> {
 }
 
 export class HttpClient {
```

Comment:
@BiswaViraj Since these `/lib/shared` files are just used by `@novu/client` package, let's move them under `packages/client` to simplify the SDK setup and have all its code collocated. Then, we will have a single, isomorphic HTTP client for Novu API, and we will be golden!